README.md

```diff
@@ -204,6 +204,15 @@ Phantoms will no longer spawn if the hostile mobcap is full. This is per player.
 - Required options: `true`, `false`
 - Categories: `FEATURE`, `CARPET-ADDONS-NOT-FOUND`
 
+### pickFlowersFromPots
+
+Right-clicking on a flowerpot with a flower in it will put the flower into your invantory.
```

Comment:
Description should be "Right-clicking on a flowerpot containing a flower will always put the flower into your inventory."